constants/rpc.ts

```diff
@@ -64,6 +64,11 @@ export const EVM_RPC_URL_MAP: Record<SUPPORTED_CHAIN_ID, string> = addAPIKey({
     process.env.SSR_RPC_BINANCE_TESTNET ||
     process.env.NEXT_PUBLIC_RPC_BINANCE_TESTNET ||
     `https://binance-testnet.${RPC_ENV}.thirdweb.com`,
+  // Question: what URL should we use for Hardhat and Localhost?
+  // temporary data for creating a build
+  // it has to be an actual URL - otherwise build will fail
+  [ChainId.Hardhat]: `https://goerli.${RPC_ENV}.thirdweb.com`,
+  [ChainId.Localhost]: `https://goerli.${RPC_ENV}.thirdweb.com`,
```

Comment:
@jnsdls need your input here, what should be RPC URL for localhost and hardhat here?
or should we remove them from this object?

Reply:
This object should no longer be necessary and we should instead rely on the configured chains (and our chains package includes localhost config)